examples/nextjs/components/HeroBanner/HeroBanner.js

```diff
@@ -1,21 +1,16 @@
 import React from 'react';
+import Image from 'next/image';
 
-import { Image } from '@nacelle/react-components';
 import * as styles from './HeroBanner.styles';
 
-const IMAGE_FORMATS = ['webp', 'jpg'];
-
 const HeroBanner = ({ fields }) => {
-  const image = fields.featuredMedia.fields;
+  const fileUrl = fields?.featuredMedia?.fields?.file?.url;
+  const alt = fields?.backgroundAltTag;
+  const src = fileUrl?.startsWith('https:') ? fileUrl : `https:${fileUrl}`;
```

Comment:
I have to admit that I am not a fan of `.?` 😖 
I'll probably get over it eventually. lol

Reply:
You're the first person I've heard that from! Tell me why some time.

Reply:
I don't like it because I think it promotes poor habits (similar to `class`). If you had something like:

```
const user = { id: '1234', name: 'Bacon Burrito', address: { street: '54 Runny Egg Ln', zip: 72703 } };
```

Typically, we'd do something like:

```
const { zip } = user.address;
```

But what if `address` is `null`? In the case of data that _we_ control, that likely means something is wrong and the error would be beneficial to see. 

However, with optional chaining, that address would silently fail and return `undefined`. This problem is the same using the more traditional syntax as well.

```
const zip = user.address && user.address.zip; // undefined
const zip = user?.address?.zip; // undefined

if (!zip) {...}
```

In the above example, there's not much to be gained from optional chaining outside of the sugar-y syntax (which isn't much shorter for this example). We still have to know that it could be `undefined` and to check for that, so that makes optional chaining _most_ useful when a property should & _CAN_ potentially be `null` (mostly from data out of our control). However, I feel like this syntax will get abused (much like `class` when it was first added) and used incorrectly (i.e. everywhere). lol - sorry for the novel

Reply:
That makes good sense! I agree that it will be used in a way that leads to silent failure when loud failure would be preferable.

> This problem is the same using the more traditional syntax as well.

100%. It's a shame that it's so difficult to enforce thoughtful destructuring / property access with proper error handling.